crates/kitsune_p2p/types/src/config.rs

```diff
@@ -503,3 +509,12 @@ pub enum TransportConfig {
     /// (this is mainly for testing)
     Mem {},
 }
+
+// Custom Url2 schema
+fn url2_schema(_: &mut SchemaGenerator) -> Schema {
```

Comment:
Should this be pulled out into another crate since its duplicated in 2 crates? (its a tiny bit of code so understandable if not worth the hassle)

Reply:
Yes, minimal code for now, but we can move it to it's own crate if we end up encountering this same issue when creating other json schemas